IDE/IAR-EWARM/embOS/user_settings.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _EMBOS_USER_SETTINGS_H_
+#define _EMBOS_USER_SETTINGS_H_
+
```

Comment:
Yes this was tested. It's pulled in through the IDE and is documented in the README I'm generating for the customer